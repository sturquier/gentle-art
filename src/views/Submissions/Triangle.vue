<template>
  <h1>Triangle</h1>
</template>
