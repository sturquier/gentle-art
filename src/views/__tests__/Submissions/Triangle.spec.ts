import { describe, it, expect } from 'vitest'
import { mount } from '@vue/test-utils'

import Triangle from '../../Submissions/Triangle.vue'

describe('Triangle', () => {
  it('renders properly', () => {
    const wrapper = mount(Triangle)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
