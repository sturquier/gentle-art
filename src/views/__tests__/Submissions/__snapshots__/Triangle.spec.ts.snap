// Vitest Snapshot v1

exports[`Triangle > renders properly 1`] = `"<h1>Triangle</h1>"`;
